VAMobile/src/screens/HealthScreen/SecureMessaging/StartNewMessage/StartNewMessage.tsx

```diff
@@ -223,6 +216,12 @@ function StartNewMessage({ navigation, route }: StartNewMessageProps) {
     setTo(undefined)
   }
 
+  useEffect(() => {
```

Comment:
UseEffect to handle single care system value setting once recipientsResponse has loaded.